src/composable/pollApi.js

```diff
@@ -0,0 +1,44 @@
+import { reactive ,computed} from "vue";
+import { useStore } from "vuex";
+export const pollApi = () => {
+  const store = useStore();
+  const pollData = reactive({
+    title: "",
+    option: []
+  });
+  
+  //get poll list
+  const polls = computed(() => {
+    return store.state.polls;
+  });
```

Comment:
if this is the case where you are not doing anything with the computed property or the assigned computed property, you can directly write them into return object like

return {
polls: computed(() => {
    return store.state.polls;
  })
}

Reply:
@mukeshsin